docs/README.md

```diff
@@ -0,0 +1,35 @@
+# Documentation
```

Comment:
@wanghaoran1988 would it be helpful to add this to README.md on the root of the repo?